### PR TITLE
feat(strm-1975): update data contract docs for fieldMetadata

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: docs/02-concepts/02-data-contracts/index.md [hunk 1]
```diff
@@ -209,30 +226,36 @@ using: `strm get data-contract strmprivacy/clickstream/1.0.0 -ojson | jq '.dataC
...
-  data subjects from each other. Typically, this is a _customer id_ or a _user id_. This is used by the Data Subjects
-  API to keep [facilitate an interface](./04-data-subjects.md) that allows for easy retrieval of used `keyLink`s for a
-  specific data subject.
+   _tie_ events together. Typically, this is what determines a data
+   subject's (i.e. your users) session. If multiple events contain the same value for
+   the field that was specified as `keyField`, then
+   the [same encryption key and `keyLink`](docs/02-concepts/01-data-processing/01-pii-field-encryption.md) will be used
+   to
+   encrypt the PII data.
+6. `piiFields`: deprecated in favor of `fieldMetadata`
```
Comment (trietsch) [inline docs/02-concepts/02-data-contracts/index.md:243]: Maybe you can create a new chip CSS class to mark this as deprecated and put some more attention to it (use `<div class="chip-optional"> <div class="chip-content">optional</div> </div>` inline to add a chip to this line).

File: docs/02-concepts/02-data-contracts/index.md [hunk 2]
```diff
@@ -209,30 +226,36 @@ using: `strm get data-contract strmprivacy/clickstream/1.0.0 -ojson | jq '.dataC
...
+7. `fieldMetadata`: description of the fields that should either be considered non-sensitive, QI or PII with
```
Comment (trietsch) [inline docs/02-concepts/02-data-contracts/index.md:244]: Please add QI to the definitions document if it's not yet there.

File: docs/02-concepts/02-data-contracts/index.md [hunk 3]
```diff
@@ -209,30 +226,36 @@ using: `strm get data-contract strmprivacy/clickstream/1.0.0 -ojson | jq '.dataC
...
+   corresponding purpose level. This field identifies if a field should be encrypted.
```
Comment (trietsch) [inline docs/02-concepts/02-data-contracts/index.md:245]: It's unclear when the field will be encrypted. Under what conditions (i.e. PII = true + purpose level is filled out).

**Review by trietsch — APPROVED** [review]

File: docs/02-concepts/02-data-contracts/index.md [hunk 4]
```diff
@@ -20,17 +20,18 @@ The Data Contract defines:
   verifications that should be done for the received content.
 
 A Data Contract is composed
-of [various elements](https://github.com/strmprivacy/api-definitions/blob/master/protos/strmprivacy/api/entities/v1/entities_v1.proto#L711)
+of [various elements](https://github.com/strmprivacy/api-definitions/blob/master/protos/strmprivacy/api/entities/v1/entities_v1.proto#L762)
```
[the remark this answers is not in the document]
Comment (bobvandenhoogen) [reply docs/02-concepts/02-data-contracts/index.md:23]: That should indeed be done. Not in the scope of this mr though.

**Pull request merged**